Resources/Prototypes/_Goobstation/Entities/Structures/Machines/material_silo.yml

```
@@ -0,0 +1,31 @@
+- type: entity
+  id: MaterialSilo
+  parent: [ BaseMachinePowered, ConstructibleMachine ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)